addons/website_event/views/event_templates_list.xml

```diff
@@ -345,7 +345,8 @@
                             </small>
                         </div>
                         <!-- Location -->
-                        <div class="d-flex align-items-center">
+                        <t t-set="event_city" t-value="event.address_id.sudo().city"/>
+                        <div t-if="not event.address_id or event_city" class="d-flex align-items-center">
```

Comment:
So if you just want to say that an event takes place in Belgium, then we don't show "Belgium" at all?

Kind of a niche use case but seems strange to hide that.